### DISCUSSION TITLE
Workflow notifications

### DISCUSSION THREAD

**Discussion by Mohinder659** [community/community#11355] [Actions]
We have set up a workflow runs-on: macos-latest. We have a team that manages the failure of deployment. But as per GitHub documentation, only the person who commits to the workflow action file will receive the notifications of Action status. The committed person may change time to time. Is there any way we can set up slack channel notifications for Github actions or email alerts only for workflow actions to multiple accounts?

We tried third party resource like https://www.ravsam.in/blog/send-slack-notification-when-github-actions-fails/#using-notify-slack-action but they only support linux.

**Comment by gr2m** [comment]
I keep running into the same problem. I work on a lot of GitHub Actions that are consumed by other repositories, where workflow runs are triggered by users who create issues / comment / change labels / etc. When a build fails, only they get notified. But as the responsible person / team behind these actions, I'd like to subscribe to at all builds that fail for specific workflows within a scope of a repository.

**Comment by ethomson** [comment]
Thanks for the feedback @Mohinder659 - I'll make sure that both the Actions team is looking into this, as well as the integrations team that works on Slack notifications.

**Reply by dhimmel** [reply]
Awesome! Similar to how you can "Watch" a repository, it'd be nice if a user could watch an action to either get notified every time it runs or just on failures.

**Reply by dhimmel** [reply]
@andrewakim I found this discussion that I had commented on in the past regarding actions notifications.

Also linking two related issues:

- general / multi-user notifications https://github.com/orgs/community/discussions/18039
- scheduled workflow notifications https://github.com/orgs/community/discussions/25351

**Reply by saivamsibodavulaTR** [reply]
@gr2m , I too have the same concern, so i tried with a different plugin in the marketplace, @ [sendmail](https://github.com/marketplace/actions/send-email) but i too have some limitations at assigning a user to the work.
